test/Plutus/Time.purs

```diff
@@ -180,8 +189,49 @@ eraSummariesFixture = EraSummaries
           , "safeZone": mkSafeZone 129600
           }
       }
+  , EraSummary
```

Comment:
The intention here is to add a fixture for fractional values but maybe it needs more.

Reply:
Maybe we could get some from Plutip?

Reply:
Do you mean in the same way we get `Ogmios` fixtures in nix or just go look in plutip repo and copy some here?

Reply:
I meant we could just make some fixtures by recording responses from a node started by Plutip. However, I don't think it should hold up this PR since you've included at least one fixture. We can open another issue and add more fixtures later

Reply:
I don't think we can replicate the Ogmios testing approach here. We can do that because the Ogmios repo has a lot of test vectors that we can use. I don't think there's anything similar for Plutip